### PR TITLE
Add geocode

### PR DESCRIPTION
**Pull request by cristineguadelupe**
https://github.com/livebook-dev/kino_maplibre/assets/5660941/f191121b-18e4-451e-8c65-ed8ca932a206

### CODE REVIEW

**Review by jonatanklosko — APPROVED** [review]
🌴 🐈

File: lib/assets/maplibre/main.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
 import "https://cdn.jsdelivr.net/npm/maplibre-gl@3.6.2/dist/maplibre-gl.min.js";
 // TODO: point to npm when released
 import "https://cdn.jsdelivr.net/gh/jimmyrocks/maplibre-gl-vector-text-protocol@main/dist/maplibre-gl-vector-text-protocol.min.js";
+import "https://unpkg.com/@maplibre/maplibre-gl-geocoder@1.2.0/dist/maplibre-gl-geocoder.min.js";
```
Comment (josevalim) [inline lib/assets/maplibre/main.js:4]: Ideally we would depend on fewer CDNs, to have fewer points of failure. I assume this is not on jsdelivr though? :(

**Review by josevalim — APPROVED** [review]
Sweet!

File: lib/assets/maplibre/main.js (hunk 1 above)
Comment (jonatanklosko) [reply lib/assets/maplibre/main.js:4]: We can esbuild, as we've done in kino!

**Pull request merged**